src/pages/help-centre/dmt5.js

```diff
@@ -42,9 +43,9 @@ const WhatIsDMT5 = () => (
 )
 const DifferenceDMT5DTrader = () => (
     <ArticleWrapper>
-        <Header as="h4" margin=" 0 0 2.4rem 0" lh="1.5">
+        <StyledHeader as="h4">
```

Comment:
here we have `lh='1.5` in Header after using StyledHeader we did not add `lh` to the component, please make sure that line-height is added to the StyledHeader

Reply:
i put default lh for header is 1.5